actionpack/lib/action_dispatch/http/parameters.rb

```diff
@@ -77,7 +77,6 @@ def path_parameters=(parameters) #:nodoc:
       end
 
       # Returns a hash with the \parameters used to form the \path of the request.
-      # Returned hash keys are strings:
```

Comment:
I see a left over. To be removed.

Reply:
I'm confused. Do you mean you need to add this back or this was left over in a previous PR? It looks like from the example below that the keys are strings.

Reply:
Sorry I mean, I am going to remove this hunk in the next rebase. Let's do this.